QSB/Instruments/InstrumentsManager.cs

```diff
@@ -0,0 +1,44 @@
+﻿using QSB.Animation;
+using QSB.EventsCore;
+using QSB.Instruments.QSBCamera;
+using QSB.Player;
+using UnityEngine;
+
+namespace QSB.Instruments
+{
+    public class InstrumentsManager : MonoBehaviour
+    {
+        public static InstrumentsManager Instance;
+        private AnimationType _savedType;
+
+        private void Awake()
+        {
+            Instance = this;
+            gameObject.AddComponent<CameraManager>();
+        }
+
+        private void Update()
+        {
+            if (Input.GetKeyDown(KeyCode.Keypad9))
```

Comment:
for debugging? 
the if could be inverted, and the key could be extracted to a constant.